etc/ProdOfflineConfiguration.py

```diff
@@ -639,7 +645,21 @@
                dqm_sequences=["@common"],
                scenario=ppScenario)
 
-# Will be "JetMET"  after  HLT V1.3 menu
+DATASET ["JetMET"]
```

Comment:
Isn't an '=' missing here?

Reply:
yes indeed, i'll update it soon! thanks for spotting this!

Reply:
done in 130f09f